packages/expo-splash-screen/android/src/main/java/expo/modules/splashscreen/SplashScreen.kt

```diff
@@ -0,0 +1,85 @@
+package expo.modules.splashscreen
+
+import android.app.Activity
+import android.util.Log
+import org.unimodules.core.interfaces.SingletonModule
+import java.util.*
+
+object SplashScreen: SingletonModule {
+  private const val TAG = "SplashScreen"
+
+  override fun getName(): String {
+    return "SplashScreen"
+  }
+
+  private val controllers = WeakHashMap<Activity, SplashScreenController>()
```

Comment:
Is this `static`? How come you're accessing it from within a `static` method?

Reply:
THere's no `static` in Kotlin. Instead there are:
- [companion objects](https://kotlinlang.org/docs/reference/object-declarations.html#companion-objects)
- and [object](https://kotlinlang.org/docs/reference/object-declarations.html#object-declarations)